chapter02/drjt/ex1.pl

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/perl
+use 5.010;
+use warnings;
+use strict;
+use File::Spec;
+use Cwd;
+
+say "    " . File::Spec->catfile(cwd() , $_) while <*>;
```

Comment:
does this while work? I'm guessing that the * looks at the pwd and not the keyboard for input